fix: all default values can be set by config

quintesse opened the pull request:
Some default values were not settable by config, an oversight.
Also added the possibility to globally set default values for all commands that support the same option.

Fixes #1576

maxandersen commented:
What does:
"Also added the possibility to globally set default values for all commands that support the same option." Refer to?

quintesse commented:
> What does: "Also added the possibility to globally set default values for all commands that support the same option." Refer to?

See the commit where I changed the docs
